Origin: http://sources.debian.net/data/main/c/cciss-vol-status/1.11-3/debian/patches/new-ids
Description: Add new PCI IDs and update names
Author: Christian Hofstaedtler <zeha@debian.org>
Forwarded: no
Last-Update: 2014-07-05

Index: cciss-vol-status/cciss_vol_status.c
===================================================================
--- cciss-vol-status.orig/cciss_vol_status.c
+++ cciss-vol-status/cciss_vol_status.c
@@ -139,15 +139,12 @@ struct smartarray_id_t {
 	{ 0x3223103C, "Smart Array P800",	0, 1},
 	{ 0x3237103c, "Smart Array E500",	0, 1},
 	{ 0x323D103C, "Smart Array P700m",	0, 1},
+	/* start of hpsa list */
 	{ 0x3241103C, "Smart Array P212",	0, 1},
 	{ 0x3243103C, "Smart Array P410",	0, 1},
 	{ 0x3245103C, "Smart Array P410i",	0, 1},
 	{ 0x3247103C, "Smart Array P411",	0, 1},
 	{ 0x3249103C, "Smart Array P812",	0, 1},
-	{ 0xe0110e11, "HP MSA500",		1, 0}, /* aka Smart Array CL */
-	{ 0xe0200e11, "HP MSA500 G2",		1, 0},
-	{ 0xe0300e11, "HP MSA20",		1, 0},
-	{ 0x3118103c, "HP B110i",		0, 1},
 	{ 0x324A103C, "Smart Array P712m", 0, 1},
 	{ 0x324B103C, "Smart Array P711m", 0, 1},
 	{ 0x3350103C, "Smart Array P222", 0, 1},
@@ -177,17 +174,30 @@
 	{ 0x21BE103C, "Smart Array P741m", 0, 1},
 	{ 0x21BF103C, "Smart HBA H240ar", 0, 1},
 	{ 0x21C0103C, "Smart Array P440ar", 0, 1},
+	{ 0x21C1103C, "Smart Array P840ar", 0, 1},
 	{ 0x21C2103C, "Smart Array P440", 0, 1},
 	{ 0x21C3103C, "Smart Array P441", 0, 1},
+	{ 0x21C4103C, "Smart Array", 0, 1},
 	{ 0x21C5103C, "Smart Array P841", 0, 1},
 	{ 0x21C6103C, "Smart HBA H244br", 0, 1},
 	{ 0x21C7103C, "Smart HBA H240", 0, 1},
 	{ 0x21C8103C, "Smart HBA H241", 0, 1},
+	{ 0x21C9103C, "Smart Array", 0, 1},
 	{ 0x21CA103C, "Smart Array P246br", 0, 1},
 	{ 0x21CB103C, "Smart Array P840", 0, 1},
 	{ 0x21CC103C, "Smart Array P542t", 0, 1},
 	{ 0x21CD103C, "Smart Array P240tr", 0, 1},
 	{ 0x21CE103C, "Smart HBA H240tr", 0, 1},
+	{ 0x00761590, "HP Storage P1224 Array Controller", 0, 1}, /* VID = 3PAR */
+	{ 0x00871590, "HP Storage P1224e Array Controller", 0, 1},
+	{ 0x007D1590, "HP Storage P1228 Array Controller", 0, 1},
+	{ 0x00881590, "HP Storage P1228e Array Controller", 0, 1},
+	{ 0x333f103c, "HP StorageWorks 1210m Array Controller", 0, 1},
+	/* end of hpsa list */
+	{ 0xe0110e11, "HP MSA500",		1, 0}, /* aka Smart Array CL */
+	{ 0xe0200e11, "HP MSA500 G2",		1, 0},
+	{ 0xe0300e11, "HP MSA20",		1, 0},
+	{ 0x3118103c, "HP B110i",		0, 1},
 
 
 #ifdef HAVE_SCSI_SG_H
